README.md

```diff
@@ -18,8 +35,6 @@ Download tool for AnVIL/TDR data identified by DRS URIs and Google
 
```

Comment:
* Add instructions on how to become a member of our test terra project: https://anvil.terra.bio/#workspaces/anvil-stage-demo/DRS-downloader/data](https://anvil.terra.bio/#workspaces/anvil-stage-demo/DRS-downloader/data

* Add instructions or a link to gcloud setup
* Add instructions to link to the `NHLBI BioData Catalyst Framework Services`.   including `=google` hack